### PR TITLE
Fix implicit-fallthrough warning

### PR DESCRIPTION
**Pull request by jayfoad**
Clang warned:
VertexFetch.cpp:1471:3: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]

Caused by "Support 'R8G8B8' vertex attribute formats"

### CODE REVIEW

**Comment by trenouf** [comment]
Snap! #2326

**Review by trenouf — APPROVED** [review]
LGTM (so you can merge yours if I forget to merge mine when its tests have finished).

**Comment by jayfoad** [comment]
Nah, I'll leave it to you.

**Pull request closed without merging**
